h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -113,7 +113,7 @@ private void writelnkv(String key, String value, boolean escapeNewlines) {
     tmpfile.append('\n');
   }
 
-  private void writelnkv(String key, String value) {
```

Comment:
Is this still needed?

Reply:
@michalkurka I think yes, I'm using it in TargetEncoderMojoWriter and for that I need to expose it via `protected` modifier.